Contributors.md

```
@@ -1,15 +1,15 @@
 ## Contributors
 
 By having yourself in the table below, all your contributions to this project
 are made under the terms of the [Developer Certificate of Origin](DCO.md).
 
-| Name                             | Github Username    | Discord Username   | Email Address                |     
-| -------------------------------- | ------------------ | ------------------ | ---------------------------- |
-| Bruno Woltzenlogel Paleo         | @Zahnentferner     | @b.wp              | zahnentferner@gmail.com      |
-| TODO                             | TODO               | TODO               | TODO                         |
+| Name                     | Github Username | Discord Username | Email Address           |
+| ------------------------ | --------------- | ---------------- | ----------------------- |
+| Bruno Woltzenlogel Paleo | @Zahnentferner  | @b.wp            | zahnentferner@gmail.com |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)